From d047ec1ae0d9d87570755ae87262264848af0289 Mon Sep 17 00:00:00 2001
From: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
Date: Mon, 22 Jun 2020 19:23:26 +0530
Subject: [PATCH 1396/1852] spi: spi-cadence: Fix uninit_use coverity warning

commit d50b23d4b5c30b72bec13ff1ddf778350b4e2f99 from
https://github.com/Xilinx/linux-xlnx.git

Fix "Event uninit_use: Using uninitialized value "num_cs""
coverity warning

Signed-off-by: Amit Kumar Mahapatra <amit.kumar-mahapatra@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/spi/spi-cadence.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/spi/spi-cadence.c b/drivers/spi/spi-cadence.c
index e0e4950b1604..792bec14e05d 100644
--- a/drivers/spi/spi-cadence.c
+++ b/drivers/spi/spi-cadence.c
@@ -475,7 +475,7 @@ static int cdns_spi_probe(struct platform_device *pdev)
 	int ret = 0, irq;
 	struct spi_master *master;
 	struct cdns_spi *xspi;
-	u32 num_cs;
+	u32 num_cs = 0;
 
 	master = spi_alloc_master(&pdev->dev, sizeof(*xspi));
 	if (!master)
-- 
2.31.1

